.devcontainer/Dockerfile

```diff
@@ -16,27 +16,22 @@ ARG  NODE_VERSION="none"
 ENV  NVM_DIR=/usr/local/share/nvm
 ENV  NVM_SYMLINK_CURRENT=true \
      PATH=${NVM_DIR}/current/bin:${PATH}
-COPY library-scripts/node-debian.sh /tmp/library-scripts/
+COPY .devcontainer/library-scripts/node-debian.sh /tmp/library-scripts/
```

Comment:
should we just copy all of the scripts at once?  each copy creates a layer and the files are small

Reply:
i think that is a good idea for our case because we're going to use all the files in that directory. i'll update it.